Makefile.core.mk

```diff
@@ -385,6 +385,15 @@ gen-api-docs: ## Generate API documentation.
 		--renderer=markdown \
 		--output-path=$(OUTPUT_DOCS_PATH) \
 		--output-mode=group
+	@echo "The following replaces are applied to the generated markdown files as workaround"
```

Comment:
I would just add this and the following two lines as comments in the Makefile. It doesn't have to be shown every time we generate the docs IMO